### PR TITLE
[feature1] first feature was implemented

### PR DESCRIPTION
**Pull request by dinkarjoshi**
Following changes were made:

- added few files
- feature1 folder was added.

### CODE REVIEW

**Comment by dinkarinjosh** [comment]
test comment

File: myfile.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is a file created in feature 1 file
```
Comment (dinkarinjosh) [inline myfile.txt:1]: Good job with the first lin (Comment)

File: myfile.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+to see the pull requet in action
```
Comment (dinkarinjosh) [inline myfile.txt:3]: i think t in to should be capital

**Review by dinkarinjosh — APPROVED** [review]
Final

File: myfile.txt (hunk 2 above)
Comment (dinkarinjosh) [inline myfile.txt:3]: two lines selected. good job

File: myfile.txt (hunk 1 above)
Comment (dinkarjoshi) [reply myfile.txt:1]: thanks

File: myfile.txt (hunk 2 above)
Comment (dinkarinjosh) [reply myfile.txt:3]: thanks for the resolution

**Comment by dinkarinjosh** [comment]
closed eevethhing in the first pull request. this is just a test comment
\

**Pull request merged**